src/stream/src/from_proto/sink.rs

```diff
@@ -296,3 +322,64 @@ impl ExecutorBuilder for SinkExecutorBuilder {
         Ok((params.info, exec).into())
     }
 }
+
+struct JdbcUrl {
+    host: String,
+    port: u16,
+    db_name: String,
+    username: String,
+    password: String,
+}
+
+fn parse_jdbc_url(url: &str) -> anyhow::Result<JdbcUrl> {
+    if !url.contains("jdbc:postgres") {
+        bail!("invalid jdbc url")
+    }
+
+    let url = url.replace("jdbc:", "");
+    let url = Url::parse(&url).map_err(|e| anyhow!(e).context("failed to parse jdbc url"))?;
```

Comment:
```suggestion
    if !url.starts_with("jdbc:") {
        bail!("invalid jdbc url")
    }

    let url = Url::parse(&url[5..]).map_err(|e| anyhow!(e).context("failed to parse jdbc url"))?;
```

Reply:
Replace jdbc is a lot more explicit, than trimming the string after 5 chars.